src/Entity/TaskSetInterface.php

```diff
@@ -12,7 +12,7 @@ interface TaskSetInterface extends TaskInterface
     /**
      * @param array<string> $tags
      *
-     * @return array<\SprykerSdk\SdkContracts\Entity\TaskInterface>
+     * @return array<\SprykerSdk\SdkContracts\Entity\TaskInterface|string>
```

Comment:
Why return type is changed? 
For me it is a bit tricky, that method can return objects or strings

Reply:
yes. because you can use task id from existing tasks. with object way it's not possible